SOURCES/aacraid-1.2.1.60001-fix-Series-6.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/comminit.c b/comminit.c
```

Comment:
Anything before this line will be seen as a comment. Please add some text explaining what the problem is and what this does to fix it.